modules/apps/auxano-theme/src/templates/portal_normal.ftl

```diff
@@ -0,0 +1,188 @@
+				<hr class="new-arrivals-break">
+				<a href="#"><h3 class="new-arrivals-link">New Arrivals</h3></a>
+			</div>
+		</div>
+
+
+
+
+		<!-- Coffee Tutorial Cards -->
+		<div class="card-container">
+
+			<div class="card">
+				<div id="image-learn" class="card-image">
+					
+				</div>
+				<div id="card-description">
+					<a href="#"><h5><br>Hario v60 Guide</h5></a>
+					<p>The Hario V60 method we at Auxano use to brew coffee from our fresh roasted beans.</p>
+					<a href="#"><h6>Learn</h6></a>
+				</div>
+			</div>
+
+			<div class="card">
+				<div id="image-blog" class="card-image">
+					
+				</div>
+				<div id="card-description">
+					<a href="#"><h5>Grinder Settings<br>Fine vs Coarse</h5></a>
+					<p>Our findings on how the coarseness of our grounds affect the taste of coffee.</p>
+					<a href="#"><h6>Blog</h6></a>
+				</div>
+			</div>
+
+			<div class="card">
+				<div id="image-event" class="card-image">
+					
+				</div>
+				<div id="card-description">
+					<a href="#"><h5><br>Latte Art Class</h5></a>
+					<p>Come join us for an evening to perfect your latte art skills with our world-class baristas.</p>
+					<a href="#"><h6>Event</h6></a>
+				</div>
+			</div>
+
+			<div class="card">
+				<div id="image-about" class="card-image">
+					
+				</div>
+				<div id="card-description">
+					<a href="#"><h5><br>Welcome Zach</h5></a>
+					<p>We'd like to introduce you to the newest member of our family, Zach!</p>
+					<a href="#"><h6>About</h6></a>
+				</div>
+			</div>
+		</div>
+
+
+
+
+		<!-- Email Sign-Up Section -->
+		<div class="email-subscription-container">
+			<div class="email-subscription-text">
+				<H2 id="newsletter">Auxano Newsletter</H2>
+				<p id="newsletter-description">We're constantly learning and growing and we want to share it with you.
+				Be the first to know about our newest releases and findings.</p>
+				<div id="email-subscription-links">
+					<a href="#"><h6 id="color-777">Email Address</h6></a>
+					<a href="#"><h6>Subscribe</h6></a>
+				</div>
+				<hr class="newsletter-break">
+			</div>
+		</div>
+
+
+
+		<!-- Footer Contact -->
+		<div class="info-footer-container">
+			<ul id="info-column-1">
+				<li><a href="#"><h3>Home</h3></a><br></li>
+				<li><a href="#"><h3>Shop</h3></a><br></li>
+				<li><a href="#"><h3>Learn</h3></a><br></li>
+				<li><a href="#"><h3>About</h3></a><br></li>
+			</ul>
+			<ul id="info-column-2">
+				<li><h3>Hours</h3><br></li>
+				<li><h3>Monday - Friday<br><span id="list-body-font">8am - 10pm</span></h3></li>
+				<li><h3>Saturday - Sunday<br><span id="list-body-font">10am - 9pm</span></h3></li>
+			</ul>
+			<ul id="info-column-3">
+				<li><a href="#"><h3>Auxano Coffee Roasters<br></h3></a></li>
+				<li><a href="#"><h3>1400 Montefino Ave<br>Diamond Bar, CA 91765</h3></a></li>
+				<li><a href="#"><h3>+1 800 284 1923<br></h3></a></li>
+				<li><a href="#"><h3>contact@auxano.com<br></h3></a></li>
+			</ul>
```

Comment:
Since all this is web content, we should allow the end user to be able to customize this content.  We can do this via theme setting (https://dev.liferay.com/develop/tutorials/-/knowledge_base/7-0/making-themes-configurable-with-settings) or an embedded portlet in the theme (https://dev.liferay.com/develop/tutorials/-/knowledge_base/7-0/embedding-portlets-in-themes-and-layout-templates).